.travis.yml

```diff
@@ -3,6 +3,7 @@ python:
 - '2.7'
 - '3.6'
 install:
+- pip install -r requirements.txt
```

Comment:
You could actually update `requirements-dev.txt` to pull it in by default, then local development benefits from the change as well :)

You can do something like this to make it happen:
```
# requirements-dev.txt
-r requirements.txt
# ... your other dev dependencies
```